src/rpc/rawtransaction.cpp

```diff
@@ -150,6 +150,60 @@ UniValue TxShieldedOutputsToJSON(const CTransaction& tx) {
     return vdesc;
 }
 
+// TODO:
+//   ephemeralKey
+//   encCiphertext
+//   outCIphertext
+//   spendAuthSig
```

Comment:
If these are documented in the RPC help but not implemented yet, then this PR should be marked as draft.

(It would be okay for these fields to be neither documented not implemented.)